include/openthread/thread_ftd.h

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2016, The OpenThread Authors.
+ *  Copyright (c) 2017, The OpenThread Authors.
```

Comment:
Given that code in this file was under the 2016 copyright, I think we should make it "2016-2017".